addons/website_sale/controllers/product_configurator.py

```diff
@@ -264,10 +264,10 @@ def _get_strikethrough_price(self, product_or_template, currency, date, price, p
             pricelist_base_price = self._apply_taxes_to_price(
                 pricelist_rule._compute_price_before_discount(
                     product_or_template,
```

Comment:
could define a product for consistency as well?
```suggestion
                    product=product_or_template,
```

Reply:
Or something like this could be added to `_compute_price_rule`:
```python
products = products or kwargs.get('product')
```
But idk if that's worth it.

Reply:
Then no, I just saw that in several places product is in `kwargs` already . Maybe it is not working correctly then :smile: 
https://github.com/odoo/odoo/blob/f30c965e62c4a60d1eceed6baf802dab3143f7b0/addons/website_sale/models/product_template.py#L289-L291

https://github.com/odoo/odoo/blob/f30c965e62c4a60d1eceed6baf802dab3143f7b0/addons/sale/models/sale_order_line.py#L595-L605

Reply:
Ah, I just double checked, and it seems like this does work because of this:

https://github.com/odoo/odoo/blob/74e5413ddf9b28c4173b211523698ced84142427/addons/product/models/product_pricelist.py#L104-L121

It takes the `product` arg, and passes it positionally to `_compute_price_rule` as its `products` arg but putting it before `*args`, so using `product=` for that one specifically should indeed be fine then.